### PR TITLE
contracts: Prepare 0.2.10 release

### PR DESCRIPTION
**Pull request by kostko**

### CODE REVIEW

File: contracts/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,18 @@ The format is inspired by [Keep a Changelog].
...
+
+ * CBOR parsing
+
+
+### Added
+
+ * `Subcall.sol` support for ROFL app authorized origin checks
+ * Initial SIWE support
+
 ## 0.2.9 (2024-07-01)
```
Comment (matevz) [inline contracts/CHANGELOG.md:21]:
```suggestion
## 0.2.9 (2024-07-30)
```

File: contracts/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,18 @@ The format is inspired by [Keep a Changelog].
 
 [Keep a Changelog]: https://keepachangelog.com/en/1.0.0/
 
+## 0.2.10 (2024-08-20)
+
+### Fixed
+
+ * CBOR parsing
+
+
```
Comment (matevz) [inline contracts/CHANGELOG.md:15]:
```suggestion
```

File: contracts/CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,18 @@ The format is inspired by [Keep a Changelog].
...
+### Added
+
+ * `Subcall.sol` support for ROFL app authorized origin checks
+ * Initial SIWE support
+
 ## 0.2.9 (2024-07-01)
 
 ### Fixed
```
Comment (matevz) [inline contracts/CHANGELOG.md:23]:
```suggestion
### Added

* Initial SIWE support

### Fixed

* Subcall undelegation done receipt
```

File: contracts/CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,18 @@ The format is inspired by [Keep a Changelog].
...
+
+### Fixed
+
+ * CBOR parsing
+
+
+### Added
+
+ * `Subcall.sol` support for ROFL app authorized origin checks
+ * Initial SIWE support
```
Comment (matevz) [inline contracts/CHANGELOG.md:19]:
```suggestion
```

**Review by matevz — APPROVED** [review]
I just realized we had a release on July 30 not 1, so SIWE and CBOR fixes already landed at that point. Additionally, the subcall undelegation done receipt was fixed then.

So the 0.2.10 release only contains your ROFL addition.

**Comment by kostko** [comment]
Please take another look.

**Pull request merged**